custom/icds_reports/tasks.py

```diff
@@ -1199,6 +1199,11 @@ def build_incentive_report(agg_date=None):
         agg_date = current_month - relativedelta(months=1)
     for state in state_ids:
         AWWIncentiveReport.aggregate(state, agg_date)
+
+    aggregate_queryset = AWWIncentiveReport.objects.all()
```

Comment:
It would be clearer if all of this data access happened in the check itself. It should not be needed to call this here and pass it in.

Reply:
there are other checks like home conduct and eligibility will use the same queryset. so i thought of creating it here and passing it as a variable. This has been refactored in https://github.com/dimagi/commcare-hq/pull/25366